core/SourceCodePos.savi

```diff
@@ -2,6 +2,10 @@
   // TODO: include more properties from the original Savi::Source::Pos?
   :var string String: ""
   :var filename String: ""
+  :: The filepath relative to the package
+  :var filepath_pkgrel String: ""
   :var dirname String: ""
+  :var pkgname String: ""
+  :var pkgpath String: ""
```

Comment:
I think I want to eventually revisit these names before Savi 1.0 - I'd like to see a more consistent and less abbreviated naming pattern.

However, this is good enough for now, and any changes we make in the future can be smoothly deprecated and later removed before Savi 1.0